src/InputWithLabel.js

```diff
@@ -0,0 +1,28 @@
+import { useRef, useEffect } from "react"
+
+function InputWithLabel({
+  todoTitle,
+  type = "text",
+  handleTitleChange,
+  children,
+}) {
+  const inputRef = useRef()
+  useEffect(() => {
+    inputRef.current.focus()
+  })
```

Comment:
Figure out how to use the `isFocused` prop in this method. The road to react book has a good example of it.